common/src/main/java/com/skytouch/common/model/Product.java

```diff
@@ -0,0 +1,90 @@
+package com.skytouch.common.model;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+
+public class Product implements Serializable {
+
+
+    private Long id;
+
+    private String name;
+
+    private String description;
+
+    private Integer unitPrice;
```

Comment:
why are you using `Integer` class instead of a primitive?
why not using `double` instead?

Reply:
you are right, double it's better, i'm going to change it.